static/app/views/monitors/components/platformPickerPanel.tsx

```diff
@@ -30,6 +32,8 @@ export const CRON_SDK_PLATFORMS: SDKPlatformInfo[] = [
   {platform: 'python', label: 'Python'},
   {platform: 'node', label: 'Node'},
   {platform: 'go', label: 'Go'},
+  {platform: 'java', label: 'Java'},
+  {platform: 'java-spring-boot', label: 'Spring Boot'},
```

Comment:
Ideally I'd like to add Quartz here as well but I don't know how best to add it. If I use `java` as platform, it takes you to the Java quickstart guide. Quartz is not a separate SDK in Java but only an integration you add ontop of any of the Java SDKs varians (e.g. Java or Spring Boot).

Reply:
Ah I think I figured it out. Only need to add the icon somewhere.

Reply:
Realize with this I need to fix the platform picker to have overflow haha 

<img width="1269" alt="image" src="https://github.com/getsentry/sentry/assets/9372512/eedce795-39c3-42ff-8fb0-cd7e4a76814a">

Let me get a PR for that first

Reply:
Also general question, do we need all these platforms to be shown here, like quartz and springboot if they don't have any upsert functionality? I know right now we have python with no upsert functionality but that is mainly to display that we support more than just celery and have an SDK for python.

I was thinking perhaps we can just add Java here for now, but keep the Quartz and Springboot quickstart docs so that if a user manually creates a cron job they will see those quartz/springboot docs in the instrumentation part of crons. Let me know if that makes sense or if you have any questions.

Reply:
> Realize with this I need to fix the platform picker to have overflow haha

Ah I didn't notice, probably because I only looked at it on a big screen.

> Also general question, do we need all these platforms to be shown here, like quartz and springboot if they don't have any upsert functionality?

I'm fine with only showing "Java" though this seems somewhat similar to picking a platform when creating a project. There we now show a modal dialog where users can select if they're using raw Java or some Framework on top of it. Maybe something similar would also fit well here?

Reply:
Looks like this:

![Screenshot 2023-10-11 at 07 09 31](https://github.com/getsentry/sentry/assets/2542832/b6e8d6bc-681f-4991-9729-018bfb66968d)

Reply:
Was about to remove them but thought maybe it's better to keep Spring Boot as that's a large portion of our Java user base. Quartz I'll remove now.

Reply:
ahhh I see now, yea that makes sense I can talk with @gaprl about this since i'm less familiar with the java landscape. We can go with springboot, and java for now. In the future I'm also fine including all three (java, quartz, springboot) if you feel that they are equally important here. But we can discuss more.

Reply:
Hmm yeah while Quartz isn't a big portion of our user base it's the standard for (more complex) scheduling in Java. So maybe it does make sense to put it back in. But we can do so in a separate PR and only do so once there's a certain set of features you'd like to see (upsert?) or once the UI is more geared towards multiple libraries per platform.

Reply:
Sounds good left a ticket so that we don't lose track of this
https://github.com/getsentry/team-crons/issues/106